server/src/models/Project/project.queries.ts

```diff
@@ -2,26 +2,41 @@
 import { prisma } from '../../../prisma/prisma-client';
 
 export const createProjectQuery = async (user_id: number, newProject: any) => {
-  console.log('yes')
-
   const project = await prisma.project.create({
-    data: newProject
+    data: newProject,
   });
 
   if (!project) throw new Error();
 
   const projectUser = await prisma.user_Projects.create({
     data: {
       user_id: user_id,
-      project_id: project.id
-    }
-  })
+      project_id: project.id,
+      id: Math.floor(Math.random() * 100),
```

Comment:
why are we creating this id? Prisma generates the id for us

Reply:
yeah it was so weird... prisma was complaining id is null...